arch/sim/src/sim/up_head.c

```diff
@@ -89,11 +89,10 @@ int main(int argc, char **argv, char **envp)
       /* Start the CPU0 emulation.  This should not return. */
 
       sim_cpu0_start();
-#else
+#endif
       /* Start the Nuttx emulation.  This should not return. */
 
       nx_start();
-#endif
     }
```

Comment:
Hmm, git rebase don't catch the confliction. I will remove it.

Reply:
Sorry, @yamt this change is related to PR, since sim_cpu0_start is compiled in host environment, we have to delcare nx_start in arch/sim/src/sim/up_simsmp.c. So it's better to avoid call nx_start inside sim_cpu0_start.